cpp/src/arrow/compute/api_scalar.h

```diff
@@ -160,6 +160,18 @@ struct ARROW_EXPORT ProjectOptions : public FunctionOptions {
 
 /// @}
 
+/// \brief Get the absolute value of a value. Array values can be of arbitrary
+/// length. If argument is null the result will be null.
+///
+/// \param[in] arg the value transformed
+/// \param[in] options arithmetic options (overflow handling), optional
+/// \param[in] ctx the function execution context, optional
+/// \return the elementwise absolute value
```

Comment:
values?

Reply:
Not sure what you refer to as "values"? The documentation is written for a single input "value", except for the comment that refers to array inputs.

Reply:
Hmm, you're right, the other docstrings are written this way.